lib/screens/stop_watch.dart

```diff
@@ -0,0 +1,301 @@
+                      fontSize: 20,
+                      fontStyle: FontStyle.normal,
+                      fontWeight: FontWeight.w600,
+                      decoration: TextDecoration.none),
+                ),
+              ],
+            ),
+          ),
+          const SizedBox(height: 48),
+          Center(
+            child: Padding(
+              padding: const EdgeInsets.fromLTRB(60, 0, 61, 0),
+              child: Text(
+                '$digitMinutes:$digitSeconds.$digitHundredths',
+                overflow: TextOverflow.ellipsis,
+                style: const TextStyle(
+                    color: Color(0xFFFFFFFF),
+                    fontFamily: 'Inter',
+                    fontSize: 64,
+                    fontStyle: FontStyle.normal,
+                    fontWeight: FontWeight.w500,
+                    decoration: TextDecoration.none),
+              ),
+            ),
+          ),
+          const SizedBox(height: 40),
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 44),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                _CircularButton(
+                  onPressed: () {
+                    (!started) ? reset() : addLaps();
+                  },
+                  data: (!started) ? 'Reset' : 'Lap',
+                  circleColor: (!started)
+                      ? const Color(0xFF1C2135)
+                      : const Color(0xFF1C2135),
+                  textColor: (!started)
+                      ? const Color(0xFF8190CD)
+                      : const Color(0xFF8190CD),
+                ),
+                _CircularButton(
+                  onPressed: () {
+                    (!started) ? start() : stop();
+                  },
+                  data: (!started) ? 'Start' : 'Stop',
+                  circleColor: (!started)
+                      ? const Color(0xFF1C3531)
+                      : const Color(0xFF351C22),
+                  textColor: (!started)
+                      ? const Color(0xFF4AE575)
+                      : const Color(0xFFDF5151),
+                ),
+              ],
+            ),
+          ),
+          const SizedBox(height: 40),
+          Expanded(
+            child: Container(
+              decoration: const BoxDecoration(color: Color(0xFF0B0D15)),
+              child: ListView.builder(
+                itemCount: laps.length,
+                itemBuilder: (context, index) {
+                  final reversedIndex = laps.length - 1 - index;
+                  final differences = calculateLapDifferences(laps);
+                  final longestDifference = differences.isNotEmpty
+                      ? differences.reduce((a, b) => a > b ? a : b)
+                      : 0.0;
+                  final shortestDifference = differences.isNotEmpty
+                      ? differences.reduce((a, b) => a < b ? a : b)
+                      : 0.0;
```

Comment:
There is no need for this to calculate every time when flutter build one item, it changes only when you add lap